Lesson1/Task3/MyQueue.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections;
+using Collections.MyLibrary;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Task3
+{
+    class MyQueue : IEnumerable<Citizen>
```

Comment:
Queue is a data structure where you can `Enqueue` `Dequeue`, other methods, as `ReturLast` for example are not needed, you can add them if you need for concrete purpose. For this case you can add other specific methods as `EnqueuePensioner`, `EnqueueCitizen`. Also you can hold last `Pentioner` to insert it easily